client/src/views/users/UsersPage.vue

```diff
@@ -137,7 +137,7 @@ const users = ref(new UserCollection());
 const currentUser: Ref<UserModel | null> = ref(null);
 
 const activeSelectedCount = computed(() => {
-  return users.value.selectedCount() - users.value.revokedSelectedCount();
+  return users.value.selectedCount();
```

Comment:
If they are the same, you can remove `activeSelectedCount` and replace it by `users.selectedCount()`

Reply:
Removed ✔️